app/elements/coder/challenge/available/register-to-challenge-dialog.html

```diff
@@ -86,20 +87,13 @@
 			},
 			_register: function(){
 				this.$.registerRequest.body = {team: this.selectedTeam ? this.selectedTeam.value : ''};
-				this.$.registerRequest.url = util.build('/challenges/' + this.challenge.canonicalName + '/register');
 				this.$.registerRequest.generateRequest();
 			},
-			_onResponse: function(){
+			_onRequestFinished: function(e){
+				e.stopPropagation();
 				this.toastText = 'Successfuly registered.';
 				this.$.toast.show();
 				this.fire('challenge-registered');
-			},
-			_onError: function(e, r){
```

Comment:
the error is handled by `coduno-iron-ajax` (handler behavior) and `errors-display` will show the error message from the platform.